pages/implementations/overview.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: default
+title: Implementation Overview
+---
+
+The rules authored in the {{site.title}} group, comes with several test cases, which are designed for easy use by test tools.
```

Comment:
I think this needs a better introduction. What is an implementation? Why is Auto-WCAG collecting information from implementors? Why would implementors want to submit their implementation?